OpenOversight/app/templates/input_find_officer.html

```diff
@@ -80,6 +80,15 @@ <h2><small>Do you remember any part of the Officer's badge number?</small></h2>
                   {% endfor %}
                 </div>
 
+                <h2><small>Do you know any part of the Officer's unique internal identifier?*</small></h2>
+                <h4><small>*This term varies by department, for example, in Baltimore it is the sequence number and in New York City it is the tax registry number.</small></h4>
+                <div class="input-group input-group-lg col-md-4 col-md-offset-4">
+                  {{ form.unique_internal_identifier(class="form-control") }}
+                  {% for error in form.badge.errors %}
```

Comment:
I think this should be `form.unique_internal_identifier.errors`, yes?

Reply:
Oy, yes, that should definitely be `form.unique_internal_identifier.errors`.  Thanks for pointing out the error.